src/fastoad/io/configuration/tests/data/conf_with_imports.yml

```
@@ -0,0 +1,29 @@
+  title: Sample OAD Process
+
+  input_file: ../results/inputs.xml
+  output_file: ../results/outputs.xml
+
+
+  imports:
+      my_driver_1: MyDriver1
+      my_driver_2: MyDriver2
```

I would suggest something like that (with the proper modules added in tests/data/...):
```suggestion
      fastoad.io.configuration.tests.data.to_be_imported.my_driver_1: MyDriver1
      "fastoad.io.configuration.tests.data.to_be_imported.my_driver_2": MyDriver2
```
It would test "complex paths" and would avoid all the boilerplate at the beginning of your test.

---

Corrected in [9d23586](https://github.com/fast-aircraft-design/FAST-OAD/pull/559/commits/9d23586af2929d71c02c34f403f2546d075f6dc7)